README.md

```diff
@@ -0,0 +1,37 @@
+Super Chemex Bot
+================
+
+Our friendly bot-overlord notifying people when there is freshly brewed coffee in the kitchen.
+
+Super Chemex Bot is a Slack app that posts a message in channel when an AWS IoT button is pressed.
+
+Setup
+=====
+
+Super Chemex Bot is only a small piece of code but requires a lot of configuration to get running. Below is a high level overview of the steps you'll have to take to get it working.
```

Comment:
sounds scary 😱

Reply:
It is 😱 

You think it's worded too strongly?

Reply:
Yeah, I'd suggest changing `a lot` to `a bit`